pkg/node/node.go

```diff
@@ -42,6 +54,38 @@ func (n *Node) Update(ctx context.Context, csi *csibaremetalv1.Deployment, schem
 		resultErr error
 	)
 
+	// in case of Openshift deployment and non default namespace - validate node service accounts security bindings
+	if csi.Spec.Platform == constant.PlatformOpenShift && csi.Namespace != constant.DefaultNamespace {
+		var rbacError rbac.Error
+		if resultErr = n.validator.ValidateRBAC(ctx, &models.RBACRules{
+			Data: &rbacmodels.ServiceAccountIsRoleBoundData{
+				ServiceAccountName: constant.NodeServiceAccountName,
+				Namespace:          csi.Namespace,
+				Role: &rbacv1.Role{
+					Rules: []rbacv1.PolicyRule{
+						{
+							Verbs:         []string{"use"},
+							APIGroups:     []string{"security.openshift.io"},
+							Resources:     []string{"securitycontextconstraints"},
+							ResourceNames: []string{"privileged"},
+						},
+					},
+				},
+			},
+			Type: models.ServiceAccountIsRoleBound,
+		}); resultErr != nil {
+			if errors.As(resultErr, &rbacError) {
+				n.eventRecorder.Eventf(ctx, csi, eventModels.WarningType, "NodeRoleValidationFailed",
```

Comment:
Failed to validate or account is not privileged?

Reply:
fix comment to 'ServiceAccount csi-node-sa has insufficient securityContextConstraints, should have privileged'